module/PowerNSX.psm1

```diff
@@ -26657,6 +26657,77 @@ function Add-NsxServiceGroupMember {
     end {}
 }
 
+function Remove-NsxServiceGroupMember {
+
+    <#
+    .SYNOPSIS
+    Removes a single Service, numerous Services, or a Service Group to a Service
+    Group
+    
+    .DESCRIPTION
+    Removes the defined Service or Service Group to an NSX Service Groups. Service
+    groups contain a mixture of selected ports to represent a potential
+    grouping of like ports.
+    
+    This cmdlet removes the defined Services or Service Groups within a Service
+    Group for specific or all Service Groups 
+    
+    .EXAMPLE
+    Get-NsxServiceGroup Heartbeat | Remove-NsxServiceGroupMember -Member $Service1
+    
+    Remove all saved Service Group Members named Heartbeat in Service Group in $Service1
```

Comment:
description is not clear.  and you probably should show how you get $service1.  Reword to something like 'Gets the service fred and removes it from the service group Heartbeat.